book/proposal/outline.md

```diff
@@ -0,0 +1,125 @@
+# Outline for Design Thinking for Developers
+
+## Chapter 1: Organic Software with Design Thinking and Domain Driven Development
+
+Organic software is the modern ideal and a quality needed if we seek to perpetuate the success and social responsibility of open source to a wider set of problems. We seek to serve the whole of life, seeking solutions which effortlessly enhance our environments in a informed, thoughtful and responsible manner.
```

Comment:
I am not sure I want to call it organic. Much of this idea is aadopted from the evolution of organic architecture as seen in the early nineteens hundreds. I like the parallel ideas and think they encompass what I am going for well. 

I would choose to focus on Domain Driven Development but I am concerned about escaping the dogmatic focus on implementation detail. I like the idea of having a parallel idea in another field to draw comparison with. How we define environment in our scenario is of vital importance however, as software is not organic in the sense of being one with the natural world instead we seek a oneness with the natural environment of the software. 

From a softwares perspective the natural environment should be a combination of the hardware environment it is running on and the problem it is solving. Bridging the gap between domain expert and computer hardware is the realm of the programming combining these two disparate worlds into a single environment is the challenge.

Reply:
The programmer is the god of this natural world, crafting the tools in which the problem solution lives. Any responsible god should focus on the constituents it is affecting more so then the tools which create such a world.

Reply:
Now I will go watch Tron 😆

Reply:
I can't believe I missed the Tron reference 😂